lib/src/app.dart

```diff
@@ -101,6 +143,8 @@ class AppConfiguration {
     this.localAppVersion,
     this.metadataEncryptionKey,
     this.metadataPersistenceMode = MetadataPersistenceMode.plaintext,
+    this.logLevel = LogLevel.error,
+    this.requestTimeout = 0,
```

Comment:
Is this intentional? Should we be setting `requestTimeout` to zero ?

Reply:
I saw the core invocation below. I think this will be misleading if one reads the values when the config is created without one. I think we should either make this nullable or copy the default value Core uses.

Reply:
There was a check in realm_core to be left by default in case it is set to 0.
But you are right, it is confusing. So that I set it to the default value which is 2 min.